pkgs/top-level/all-packages.nix

```diff
@@ -19882,6 +19882,8 @@ with pkgs;
 
   obelisk = callPackage ../development/tools/ocaml/obelisk { menhir = ocamlPackages.menhir; };
 
+  obj2voxel = callPackage ../tools/graphics/obj2voxel { };
```

Comment:
Could you also switch this to the new `by-name` structure introduced in #237439 ?

Reply:
Done, seems to work.